### ISSUE TITLE
Complete Review 

### ISSUE THREAD

**Issue by bwayne2015** [sumitmckv/code-hunt#2]
Please complete the review by EOD for python-array/list.
https://github.com/sumitmckv/code-hunt/blob/array-first-commit/data-structure/array/PYTHON/Array_in_Python.md
The further work can be started after this gets completed

**Comment by sumitmckv** [comment]
1) Note : In Python to define a variable you don't have to write it's type while defining or assigning, Python compiler takes care of it(line 14)
->for assignment in no other languages,  type declaration is not required i guess.
2)please explain how to delete the whole list.
3)please write some small doc for list.sort() and any other useful methods if missing.
4)please give some good examples where we can use list.
5)example for taking list elements from console input or from a file will be great. 
6)please explain how to define 2d or multidimensional list and make use of them.

**Comment by bwayne2015** [comment]
I will complete this points by EOD. Though the first point is  littl bit unclear as for C it is needed and for javascript we do it using VAR and if you don't put var it will consider the element s a global variable but for python such scenarios not present

**Comment by sumitmckv** [comment]
var or any type we put it before the variable name while declaration, not while assignment
var x; // declaration
x = 5; // assignment
if we are doing the both thing in the same place then its different case.
like var x = 5;
but in python that is not required.

**Comment by bwayne2015** [comment]
That was what's my point is in python declaration and assignment done together. Declaring a variable is an extra and wastage of editorspace and time.Rather do it at the same time

**Comment by bwayne2015** [comment]
Hope this article helps you understand the concept better:
http://stackoverflow.com/questions/664294/is-it-possible-only-to-declare-a-variable-without-assigning-any-value-in-python

"Python is dynamic, so you don't need to declare things; they exist automatically in the first scope where they're assigned. So, all you need is a regular old assignment statement as above.

This is nice, because you'll never end up with an uninitialized variable. But be careful -- this doesn't mean that you won't end up with incorrectly initialized variables"

**Comment by sumitmckv** [comment]
Yes it's clear now as in python  uninitialized variable is not a very good practice.
Note : In Python to define a variable you don't have to write it's type while defining or assigning
-> so can we write it like Note : In Python to define a variable you don't have to write it's type before it?

**Comment by bwayne2015** [comment]
Yes nice approach will update today.

**Comment by sumitmckv** [comment]
Please complete the job and commit your changes in master branch then close the issue.
And @aniketSinha001 haven't commented yet.
